tests/test_features/test_correction/test_generator.py

```
@@ -244,6 +244,124 @@ def test_validate_counts_only_numbered_lines_with_subheaders(monkeypatch: pytest
     assert "description의 line_number 4가 원본 라인 수(3)를 벗어났습니다." in invalid_errors
 
 
+def test_correction_yaml_max_retries_has_priority(monkeypatch: pytest.MonkeyPatch):
+    """max_retries는 correction.yaml(validation) 값을 우선 사용한다."""
+    from features.correction import generator
+
+    monkeypatch.setattr(generator, "get_llm", lambda model=None, temperature=0.2: DummyLLM())
```

Ruff (0.15.2)

[warning] Unused lambda argument: `model`

(ARG005)